### DISCUSSION TITLE
It should be possible to disable spotfy integration

### DISCUSSION THREAD

**Discussion by ScrambleTheRamble** [navidrome/navidrome#2814] [Ideas]
Currently it is possible to disable the Last.fm integration via the config key "LastFM.Enabled".  This is not possible (or not documented) for Spotify.

**Comment by deluan** [comment]
If you don't have a `Spotify.ID` / `Spotify.Secret` configured, it will be disabled and won't try to connect to Spotify. If you see something different in the logs, let me know.

**Reply by ScrambleTheRamble** [reply]
Can confirm, logs state the agent isn't available:
`Jan 22 20:16:16 navidrome navidrome[17718]: time="2024-01-22T20:16:16Z" level=info msg="Spotify integration is not enabled: missing ID/Secret"
Jan 22 20:16:16 navidrome navidrome[17718]: time="2024-01-22T20:16:16Z" level=info msg="Mounting Native API routes" path=/api
Jan 22 20:16:16 navidrome navidrome[17718]: time="2024-01-22T20:16:16Z" level=error msg="Agent not available. Check configuration" name=spotify
`
